CONTRIBUTING.md

```diff
@@ -13,40 +13,11 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## Microsoft Open Source Code of Conduct
+
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 
 Resources:
 
 - [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)
 - [Microsoft Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/)
 - Contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with questions or concerns
-
-## Browser support
-Please make sure to support all modern browsers as well as Internet Explorer 11.
```

Comment:
Do we define the browsers we support explicitly?
 
Azure portal does it here. 
[https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-supported-browsers-devices](https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-supported-browsers-devices)

O365 also explicitly calls them out.

Reply:
I'll link to the portal support. We should match that.